charts/plugin-br-pix-switch/templates/ingress-apps.yaml

```
@@ -0,0 +1,63 @@
+{{- $ctx := . }}
+{{- $cfg := .Values.appsIngress }}
+{{- $fullName := include "plugin-br-pix-switch.componentFullname" (dict "context" $ctx "component" "apps") }}
+{{- /* Collect routes whose target component is enabled. */}}
+{{- $renderedRoutes := list }}
+{{- range $route := $cfg.routes }}
+  {{- $componentValues := index $ctx.Values $route.component }}
+  {{- if and $componentValues $componentValues.enabled }}
+    {{- $renderedRoutes = append $renderedRoutes $route }}
+  {{- end }}
+{{- end }}
+{{- if $renderedRoutes }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)